File: js_project/Strings/ReverseString.py
s1 = "Abhishek"
k =3
length = len(s1)
rotatedResult = ""

for i in range (length-1, length-k-1, -1):
    rotatedResult = rotatedResult + s1[i]

for i in range (0, length-k):
    rotatedResult = rotatedResult + s1[i]
    
print("Original String: " + s1)
print("Rotated by K'th Position Result  : " + rotatedResult)

File: java_project/String/ReverseString.java
    package String;

    public class ReverseString {
        public static void main(String[] args) {
            String text = "Abhishek";
            int k = 3;
            int length = text.length();
            
            String rotatedResult = "";
            
        
            for (int i = length - 1; i >= length - k; i--) {
                rotatedResult = rotatedResult + text.charAt(i);
            }
            
            
            for (int i = 0; i < length - k; i++) {
                rotatedResult = rotatedResult + text.charAt(i);
            }
            
            // 3. Print the final result
            System.out.println("Original String: " + text);
            System.out.println("Rotated by K'th Position Result  : " + rotatedResult);
        

        }
    }
            
        

